miso-web/src/main/webapp/scripts/hot_qctype.js

```diff
@@ -0,0 +1,34 @@
+HotTarget.qctype = {
+  createUrl: '/miso/rest/qctype',
+  updateUrl: '/miso/rest/qctype/',
+  requestConfiguration: function(config, callback) {
+    callback(config)
+  },
+  fixUp: function(qctype, errorHandler) {
+  },
+  createColumns: function(config, create, data) {
+    return [
+      HotUtils.makeColumnForText('Name', true, 'name', {
+        unpackAfterSave: true,
+        validator: HotUtils.validator.requiredText
+      }), HotUtils.makeColumnForText('Description', true, 'description', {
+        unpackAfterSave: true,
+      }), HotUtils.makeColumnForEnum('Target', true, true, 'qcTarget', config.qcTargets, null),
+      HotUtils.makeColumnForText('Units', true, 'units', {
+        unpackAfterSave: true,
+      }), HotUtils.makeColumnForInt('Precision After Decimal', true, 'precisionAfterDecimal', HotUtils.validator.requiredPositiveInt),
```

Comment:
So, this is a bit complicated. We allow -1 for this value to indicate that the QC is a boolean QC. I think you'll have to write a special validator for this one. I'm not sure if there's a more friendly way to display that. Realistically, they're not going to enter more than 4 or 5 digits, so it might be worth having a fixed list of -1 through 5. @callunity, what do you think?

Reply:
Y'know, I'm not sure that we actually use that anymore other than to specify whether it's a boolean QC. No code actually makes digit modifications based on this value, and I think we decided that was better than the lab getting upset if we truncate/add significant digits (probably in response to [this issue](https://jira.oicr.on.ca/browse/GLT-1936)). I think it would be more accurate to change this entire `precisionAfterDecimal` field to be an `isBoolean` marker instead... what do you think?